shared/wallets/banner/index.tsx

```diff
@@ -48,7 +48,7 @@ const Banner = (props: Props) => (
         props.onAction && (
           <>
             {props.offerAdvancedSendForm === AdvancedBanner.receiverBanner &&
-              'This user accepts more assets than XLM. '}
+              'This person accepts other assets than XLM. '}
             {props.offerAdvancedSendForm === AdvancedBanner.senderBanner &&
               'You can send more assets than XLM. '}
```

Comment:
@songgao You may want to change this one too: "You can send assets other than XLM."